.config.template.toml

```
@@ -32,6 +32,8 @@ storymode = { optional = true, type = "bool", default = false, example = false,
 
 [settings.background]
 background_choice = { optional = true, default = "minecraft", example = "minecraft", options = ["minecraft", "gta", "rocket-league", "motor-gta", ""], explanation = "Sets the background for the video" }
+width = { optional = true, default = 1080, example = 1080, explanation = "Sets the width resolution for the video", type = "int", nmin = 1, oob_error = "The width HAS to be greater than 1", input_error = "The width HAS to be an integer greater than 1" }
+height = { optional = true, default = 1920, example = 1920, explanation = "Sets the height resolution for the video", type = "int", nmin = 1, oob_error = "The height HAS to be greater than 1", input_error = "The height HAS to be an integer greater than 1" }
```

I think 500x500 fill be small enough and it'll prevent user errors
```suggestion
width = { optional = true, default = 1080, example = 1080, explanation = "Sets the width resolution for the video", type = "int", nmin = 500, oob_error = "The width HAS to be greater than 1", input_error = "The width HAS to be an integer greater than 1" }
height = { optional = true, default = 1920, example = 1920, explanation = "Sets the height resolution for the video", type = "int", nmin = 500, oob_error = "The height HAS to be greater than 1", input_error = "The height HAS to be an integer greater than 1" }
```